public/app/features/query/components/QueryGroupOptions.tsx

```diff
@@ -371,22 +382,31 @@ export class QueryGroupOptionsEditor extends PureComponent<Props, State> {
         </div>
 
         <div className="gf-form">
-          <span className="gf-form-label width-9">Time shift</span>
+          <InlineFormLabel
+            width={9}
+            tooltip={
+              <>
+                Overrides the time range for individual panels by shifting its start and end relative to the time
+                picker.
+              </>
+            }
+          >
+            Time shift
+          </InlineFormLabel>
           <Input
             type="text"
             className="width-6"
-            placeholder="1h"
+            placeholder="0h"
```

Comment:
```suggestion
            placeholder="1h"
```

Please reverti this, it's like an example shift, 0h doesn't seem like a good example value.

Reply:
If the label here is set to `1h`, it seems possible to emphasize in the tooltip that the default value is `0h`. Otherwise, this example may be misleading to users, or they may not understand what behavior to expect when not filling in the field.

Reply:
🤔 Had another look at this, and technically the main purpose of a placeholder is to provide a sample or short description of the expected value. This means if we put `0h` is because is a valid value, but this is not currently a valid value because is not going to shift the time. I think I agree with @dprokop, let's keep the `1h`